openlibrary/plugins/openlibrary/js/availability.js

```diff
@@ -202,6 +199,7 @@ function initAvailability() {
             return;
         }
 
+        var results = $('a.results');
```

Comment:
This missing line was causing availability.js to break. Discovered by diff'ing 
https://github.com/internetarchive/openlibrary/blob/0ad94ac0435f90bd552af800ff48c40892f97490/openlibrary/plugins/openlibrary/js/availability.js